logging_config/logging_set_up.py

```diff
@@ -1,18 +1,16 @@
-import io
 import json
 import logging
 import logging.config as logging_config
 import os
 from datetime import date, datetime
-from inspect import getframeinfo, stack
 
-import colorama
```

Comment:
I don't remember the particular usage of the package. It looks like it is not in use, but it is in the dependencies list. Am I right?